tutorials/build_system/prerequisites.md

```diff
@@ -2,122 +2,201 @@
+  }
+```
 
-> **Note** Please make sure to always upgrade to latest version of `gradle-intellij-plugin` plugin, follow releases on [GitHub](https://github.com/JetBrains/gradle-intellij-plugin/releases). 
+### Opening a Wizard-Generated Gradle Project for the First Time
+The first time a project is opened an _Import Project from Gradle_ dialog appears.
+This dialog is essentially the same as the Wizard [Gradle Settings Screen](#gradle-settings-screen).
+Opening the project is an import because the Gradle plugin translates some of the project's Gradle files into IntelliJ IDEA project working files.
+For example, the import creates a `.idea` directory and a `<project_name>.iml` file.
+
+These IntelliJ IDEA project working files can be saved with the project to avoid the import step in the future.
+See the note at the end of the [Creating a new Gradle project](https://www.jetbrains.com/help/idea/gradle.html#project_create_gradle) about how newly created or imported projects are stored externally by default.
+
+## Adding Gradle Support to an Existing DevKit-Based IntelliJ Platform Plugin
+As of this writing, there isn't an IntelliJ IDEA Wizard to automatically convert a DevKit-based plugin project to a Gradle-based plugin project.
+Conversion can be done, however, using the New Project Wizard to create a Gradle-based project around the existing DevKit-based project: 
+* Ensure the directory containing the DevKit-based IntelliJ Platform plugin project can be fully recovered if necessary.
+* Delete all the artifacts of the DevKit-based project:
+  * `.idea` directory
+  * IML file
+  * `out` directory
+* Arrange the existing source files within the project directory in Gradle `SourceSet` format. 
+* Use [Gradle Plugin Wizard](#creating-a-gradle-based-intellij-platform-plugin-with-new-project-wizard) as though creating a new project from scratch.
+* On the [Project Naming Screen](#project-naming-screen) set the values to:
+  * GroupID to the existing package in the initial source set.
+  * ArtifactID to the name of the existing plugin.
+  * Version to the same as the existing plugin.
+* On the [Project Name and Location Screen](#project-project-name-and-location-screen) set the values to:
+  * _Project name_ to the name of the existing plugin. 
+    (It should be pre-filled from the _ArtifactID_)
+  * Set the _Project location_ to the directory of the existing plugin.
+* Click _Finish_ to create the new Gradle-based plugin.
+* [Add more modules](https://www.jetbrains.com/help/idea/gradle.html#gradle_add_module) using Gradle [_Source Sets_](https://www.jetbrains.com/help/idea/gradle.html#gradle_source_sets) as needed.
+
+
+## Running a Simple Gradle-Based IntelliJ Platform Plugin
+Open the IntelliJ Platform SDK in IntelliJ IDEA Ultimate or Community Edition using **File \| Open** and navigating to the root directory of the local copy of the IntelliJ Platform SDK. Open the Gradle tool window and look for the _gradle_plugin_demo_ project:
```

Comment:
> Open the IntelliJ Platform SDK

I'm not quite understand what you're suggesting here. Do you mean that user should check out sources of this repository? I think it would be rather inconvenient. I think it would be simpler to just tell how to create an action as it was done in the previous version of this page.

Reply:
I can certainly restore the previous content if you feel that approach is less work for the user. It just seemed tedious to me. All the previous section accomplished was to recreate the gradle-plugin-demo in order to demonstrate executing the runIdea task. That seemed like a lot of work for little benefit.

I assume that people do download the intellij-sdk-docs repo because it is the only way at present to build the sdk code samples. (I would like to change that - IJSDK-559.)

Reply:
I restored the previous content with a modest rewrite.